README_BUILD.md

```diff
@@ -1,45 +1,56 @@
-Building DSC Resources
-======================
-## Quick-start
+# Building DSC Resources
 
-~~~
-# install the bundler gem
-gem install bundler
+The `dsc` module uses rake tasks to parse the DSC Resource MOF schema files to generate Puppet types. This process pulls the DSC Resources from the github [PowerShell/DSCResources](https:/github.com/powershell/DSCResources) repo using git. This is a point-in-time snapshot, so anytime any of those DSC Resources are updated the build process needs to be run again to pull in the updated resources and parse them. If one wants to use non Microsoft DSC Resources, then you should follow the [Build Custom DSC Resource Types](#building-puppet-types-from-custom-dsc-resources) instructions.
 
-# install the needed ruby gems/libs
-bundle install
+## Requirements
+
+* This module must be built with a **non-Windows** computer due to limitations in the gems that the builder uses to generate types and specs.
+* Git client needs to be version 2.2.0 or above
```

Comment:
Should either always periods at the end of the bullet, or never.  Need to pick one.